src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -214,8 +224,11 @@ const SkillsQuizStepper = ({ isStyleAutoSuggest }) => {
                       searchClient={searchClient}
                     >
                       <div className="mt-4.5">
-                        Second, which industry describes where you work? (select
-                        one, or leave blank)
+                        <FormattedMessage
+                          id="enterprise.skills.quiz.v1.industry.selection.label"
+                          defaultMessage="Second, which industry describes where you work? (select one, or leave blank)"
+                          description="Industry selection label prompting the user to select their relevant industry on the skills quiz v1 page."
```

Comment:
Can we please update the description to give some context regarding what we mean by `select one, or leave blank` specifically `select one`? I think this is a label for a dropdown and the user has to either select one option from dropdown or leave blank?